src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/idconverter/CustomBackendIdConverter.java

```diff
@@ -18,19 +18,19 @@
 package uk.ac.ebi.ampt2d.metadata.persistence.idconverter;
 
 import org.springframework.core.convert.converter.Converter;
-import uk.ac.ebi.ampt2d.metadata.persistence.entities.AccessionVersionEntityId;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.AccessionVersionId;
 
-public class CustomBackendIdConverter implements Converter<String, AccessionVersionEntityId> {
+public class CustomBackendIdConverter implements Converter<String, AccessionVersionId> {
```

Comment:
With all the entities now use `long` ids, do we still need this id converter?

Reply:
it is used for querydsl queries , like below 
``` 
mockMvc.perform(get("/studies?accessionVersionId=EGAS0001")).andExpect(status().is4xxClientError())
                .andExpect(jsonPath("$" +
                ".message").value("Please provide an ID in the form accession.version")); 
``` . If we don't need these we can remove it as i guess.

Reply:
Yes, we still need queries by accession+version.